projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -0,0 +1,60 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@flag-width: 1.75rem;
+@flag-height: 1.25rem;
+
+:host {
+    display: block;
+
+    &._disabled {
+        pointer-events: none;
+    }
+}
+
+.t-country-select {
+    width: 3.625rem;
+
+    & select:not(:disabled):not(._readonly) {
```

Comment:
Wouldn't disabled be covered by pointer-events: none on the host? Also sounds like point should be built in from the tui-textfield side here, every control based on select would require that.

Reply:
Drop this line in https://github.com/taiga-family/taiga-ui/pull/7580/commits/fd43ede112379c0cd1e908839204fdd7912098ec + open the following PR:

* https://github.com/taiga-family/taiga-ui/pull/7603